nizarmah/urban-tree
# Implement basic "hello world" typescript express server

### Description

Creates initial basic hello world typescript express server.

This guide uses `nvm` along with digital ocean's tutorial - [_How To Set Up a Node Project With Typescript_](https://www.digitalocean.com/community/tutorials/setting-up-a-node-project-with-typescript).

### Testing instructions

* Follow the documentation to set things up
* Run the following command: `npm run deploy`
* Navigate to: `http://localhost:3000`
* Ensure you receive the following text: `Hello World`
